apps/backend/src/sales/dto/sales-wip.dto.ts

```diff
@@ -0,0 +1,77 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsNotEmpty,
+  IsString,
+  ArrayMinSize,
+  IsArray,
+  ValidateNested,
+  IsNumber,
+  IsDefined,
+  ArrayUnique
+} from 'class-validator';
+import { PaymentsDTO } from './payment.dto';
+import { Type } from 'class-transformer';
+
+export class SalesReconciliationDTO {
+  @ApiProperty({
+    description: 'id',
+    example: '20230103-00025-1000001'
+  })
+  @IsString()
+  @IsNotEmpty()
+  id!: string;
+
+  @ApiProperty({
+    description: 'Agent No.',
```

Comment:
Agent no is very garms specific, location of the sale might be a more generic term? Probably consult with Qudsia or Karim